spec/requests/dummy_controller_spec.rb

```diff
@@ -62,6 +62,16 @@
       expect {get '/dummy/blocked_by_block'}.to raise_error ActionController::RoutingError
     end
   end
+
+  it 'should not vomit given a list of IPs in HTTP_X_FORWARDED_FOR -- and should look at the leftmost IP in the list' do
+    ip_list = '1.2.3.4, 10.0.0.1'
+    get root_path, nil, {"HTTP_X_FORWARDED_FOR" => ip_list}
+    open_session do |sess|
+      sess.remote_addr = ip_list
```

Comment:
No, I set up this test to make sure it grabs the correct/leftmost IP as the client IP and blocks it.

10.0.0.1 would *not* raise an error in the 'blocked_by_block' whitelist.